presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,372 @@
+        return null;
+    }
+
+    public static Path createCoordinatorNodeProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("node", ".properties");
+
+            String properties =
+                    "node.environment=testing\n" +
+                            "node.location=testing-location\n" +
+                            "node.data-dir=/var/lib/presto/data"
+                            + System.lineSeparator();
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create config.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createNativeWorkerEntryPointScript()
+    {
+        try {
+            Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+            String scriptContent =
+                    "#!/bin/sh\n\n" +
+                            "GLOG_logtostderr=1 presto_server \\\n" +
+                            "    --etc-dir=/opt/presto-server/etc"
+                            + System.lineSeparator();
+
+            Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+            entryPointScript.toFile().setExecutable(true);
+            return entryPointScript;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create entrypoint.sh and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createCoordinatorEntryPointScript()
+    {
+        try {
+            Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+            String scriptContent =
+                    "#!/bin/sh\n" +
+                            "set -e\n" +
+                            "$PRESTO_HOME/bin/launcher run"
+                            + System.lineSeparator();
+
+            Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+            entryPointScript.toFile().setExecutable(true);
+            return entryPointScript;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create entrypoint.sh and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    static Path createNodeProperties(String nodeId)
+    {
+        try {
+            Path tempNodeProperties = Files.createTempFile("node", ".properties");
+
+            String nodeProperties =
+                    "node.environment=testing\n" +
+                            "node.location=testing-location"
+                            + System.lineSeparator();
+
+            Files.write(tempNodeProperties, nodeProperties.getBytes(), StandardOpenOption.WRITE);
+
+            String nodeIdEntry = "node.id=" + nodeId + System.lineSeparator();
```

Comment:
Better to use a standard line separator rather than making the output platform dependent

Reply:
Updated the code.